### ISSUE TITLE
participants shouldn't be able to edit/delete other participants/votes

### ISSUE THREAD

**Issue by ste7613** [kellerben/dudle#45]
Hi

participants shouldn't be able to edit/delete other participants/votes.
I know in doodle it is the same problem for anon user, but you can login to protect your entry.

bye
Ste

**Comment by brc111** [comment]
I don't think users can edit/delete other user's participation in a doodle polls currently. Anon or logged in.

Would much prefer our company used an independent service like dudle than google's doodle. Hope this issue can be fixed.

**Comment by logg-sar** [comment]
@brc111 

They can.

I agree to ste7613 that they shouldn´t!

**Comment by Turakar** [comment]
Maybe you can implement some kind of "protected mode" where a login or a admin password is required to authenticate you if you want to edit the poll.

**Comment by brc111** [comment]
Ah, yes. I always create 'Hidden Polls' for privacy, These can not be seen or edited by other users.

![doodle](https://user-images.githubusercontent.com/22743581/28208500-de1e4d70-6885-11e7-896b-0315a5a0b964.JPG)

I would **_love_** to use dudle for our work more but we need private & un-editable entries.

**Comment by kellerben** [comment]
This is the dudle variant of hidden polls:
https://dudle.inf.tu-dresden.de/example.cgi?poll=coffeebreak-asymcrypt

**Comment by brc111** [comment]
While I am a big fan of encryption, It's frequently not possible for employees to install PGP on work computers, in order to view the poll results.

However, yes, I see that the alternative of lockable polls, would require an entire user/password/log-in infrastructure to be set up.

**Comment by logg-sar** [comment]
@brc111
IMHO is a "entire user/password/log-in infrastructure" to much work to implement.

The easier way would be to protect the entry (==set it read only) in the moment the user saved it. 
If the entry isn't correct the user can add a new one. Only the owner/administrator should be able to edit/change/delete the entries.

What do you mean - how complicated would be this?

**Comment by brc111** [comment]
>The easier way would be to protect the entry (==set it read only) in the moment the user saved it.

Yes, http://www.propoodle.net manages it that way.  (..does unfortunately leave the user's name displayed but at least it can't be changed by other users).

**Comment by josch** [comment]
I'd also be very happy if there was an option for this. I'd like to implement it but I'm currently stuck at where to put the checkbox that controls whether or not the poll is editable by anonymous. It doesn't really fit into any of the existing tabs. Ideas?

gpg encrypted polls are great but I'm afraid that most of my friends don't even know what gpg is. So for me it would suffice if I could create a poll where, after an anonymous user clicked the submit button, they cannot change anything anymore. That the only thing they can then do is to add a new row. It is then left to the admin to do cleanup if necessary.

**Comment by logg-sar** [comment]
@josch 
What about this request?

Any chance for this change?

IMHO you dont need to implement anonymous edit.
An anonymous (or no-admin user) can add an additional entry if he entered something wrong. The admin(s) should be the only ones to delete.
